pkg/webservice/dao/partition_info.go

```diff
@@ -24,6 +24,14 @@ type PartitionDAOInfo struct {
 	Queues        QueueDAOInfo      `json:"queues"`
 }
 
+type PartitionInfo struct {
```

Comment:
Can we add some fields about the partition state?
- state
- lastStateTransitionTime